Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-1-A.yml

```diff
@@ -20,21 +20,24 @@ title:
 
 #Describe the benefit that is achieved by implementing this activity
 benefit: 
-  The risk of human errors done during deployment and leading to security issues is significantly mitigated. 
+  The risk of human errors during deployment that lead to securit=y issues is significantly reduced. 
```

Comment:
Typo (securit=y). Maybe "The risk of human errors during deployment leading to security issues is significantly reduced?" sounds a bit better to me?

Reply:
Changed to suggestion.